src/main/java/org/spongepowered/common/service/scheduler/AsyncScheduler.java

```diff
@@ -106,608 +51,139 @@
     private final Condition condition = this.lock.newCondition();
     // The dynamic thread pooling executor of asynchronous tasks.
     private ExecutorService executor;
-    // Query actor for task information
-    private SchedulerHelper schedulerHelper;
 
     private AsyncScheduler() {
```

Comment:
Going to update https://github.com/SpongePowered/SpongeAPI/pull/697, therefore this PR will be on hold until the scheduler API is finalized.